.github/workflows/code-checks.yml

```diff
@@ -0,0 +1,47 @@
+name: Dasharo Checks
+on:
+  push:
+
+jobs:
+  lint:
+    runs-on: ubuntu-22.04
+    steps:
+    - name: Checkout repository
+      uses: actions/checkout@v4
+      with:
+        # Checkout pull request HEAD commit instead of merge commit
+        # See: https://github.com/actions/checkout#checkout-pull-request-head-commit-instead-of-merge-commit
+        ref: ${{ github.event.pull_request.head.sha }}
+        # Top commit is enough
+        fetch-depth: 1
+    - name: Standard lint tests
+      run: util/lint/lint lint-extended
```

Comment:
Isn't `make test-lint` a superset of lint-extended? it seems it runs lint-stable and lint-extended:

```
Check that files have license headers (lint-stable-000-license-headers): success
Check for superfluous whitespace in the tree (lint-stable-003-whitespace): success
Check that C labels begin at start-of-line (lint-stable-004-style-labels): success
Check that every board has a meaningful board_info.txt (lint-stable-005-board-status): success
Check that every vendor and board has a Kconfig.name (lint-stable-006-board-name): success
Check Kconfig files for errors (lint-stable-008-kconfig): success
Verify that files don't have the old style header (lint-stable-009-old-licenses): success
Check that we use a single assembler syntax (lint-stable-010-asm-syntax): success
Check that source files are not executable (lint-stable-012-executable-bit): success
Verify that site-local is not in the coreboot repository (lint-stable-013-site-local): success
Check for non-ASCII and unprintable characters (lint-stable-016-non-ascii): success
Check that saved config files are miniconfigs (lint-stable-017-configs): success
Report any symbolic links (lint-stable-018-symlinks): success
Check for auto-included headers (lint-stable-019-header-files): success
Verify that the word 'coreboot' is lowercase (lint-stable-021-coreboot-lowercase): success
Run clang-format on white-listed directories (lint-stable-022-clang-format): success
Check for illegal characters in filename (lint-stable-023-filenames): success
Check that no board's Kconfig sets SUBSYSTEM_(VENDOR|DEVICE)_ID (lint-stable-024-kconfig-no-subsystem): success
Verify that files don't contain windows line endings (lint-stable-026-line-endings): success
Check that path patterns in MAINTAINERS have trailing slash (lint-stable-027-maintainers-syntax): success
Check that no touchpad uses the "probed" flag (lint-stable-028-touchpad-probed-warning): success
Run checkpatch on directories that are known good (lint-extended-007-checkpatch): success
Check that files end with a single newline (lint-extended-015-final-newlines): src/mainboard/clevo/tgl-u/acpi/touchpad.asl has multiple final newlines.
src/mainboard/msi/ms7e06/msi_id.S has multiple final newlines.
test failed
Check for a signed-off-by line on the latest commit (lint-extended-020-signed-off-by): success
```

Reply:
`lint` runs `lint-stable`, `lint-extended` and "unstable" tests which can fail:
```
Usage: ./lint <sub-command> [Options]
 Sub-commands:
    lint-stable        : Run standard lint tests - should pass
    lint-extended      : Run extended lint tests - should pass
    lint               : Run full lint tests - Not expected to pass

  Options:
    -h | --help       : Show this help message
    -I | --invert     : Invert results - used for testing linters
    -J | --junit      : Send test output to a JUnit file
```

This made me notice that I ran extended tests twice: https://github.com/Dasharo/coreboot/compare/9775ffea1314dc23a91ab9f46425791e71c8eb3c..3b9467bbf6b67cbf2b1080a59378c8779cba4c3e#diff-ddf88e15b08104435ae66be9982938335f6c290a85de4cb9a09868e0e01dd4d4